docs/pages/docs/9-api-references/ddl.md

```diff
@@ -56,6 +56,10 @@ one will be created.
 
 In the case where `$B` does not have a defined `sub`, this will also implicit define `$B sub role`.
 
+## as
+
+`$A relates $C as $D` is equivalent to `$A relates $C; $C subs $D`. It can can only be used together with `relates`. 
```

Comment:
Since this is really part of `relates`, it might be simpler to just remove the `## as` header.

```
$C subs $D
```
should be:
```
$C sub $D
```